### DISCUSSION TITLE
[Esame 27 Gennaio 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#345] [Esami - Bandiera]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/c8857d92-ff49-413c-b462-d793b9847b84)

**Comment by CiottoloMaggico** [comment]
Non garantisco la correttenza anche se i conti dovrebbero essere giusti
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/8a154c09-015e-42e9-b47b-32f51da51b1f)
